gapic/src/main/com/google/gapid/views/TracerDialog.java

```diff
@@ -520,6 +511,18 @@ protected void configureDialog(DirectoryDialog dialog) {
         colorFilledInput(widgets.theme);
       }
 
+      private void cacheDeviceIsValidated() {
```

Comment:
This is a miss-leading name, please change it. We are planning on caching device validation results between runs, so lets reserve any reference to validation caching to that.